blueprints/gitlab-ci-phantom/files/gitlab-ci.yml

```diff
@@ -0,0 +1,13 @@
+image: node:latest
+
+before_script:
+  - npm config set spin false
+  - npm install -g bower
+  - npm install -g ember-cli
+  - npm install -g phantomjs-prebuilt
+  - npm install
+  - bower install --allow-root
+
+test:
+  script:
+    - ember test
```

Comment:
I'd love to see a default setup for the new YAML anchors stuff https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/2958, letting users potentially run more than one ember-try scenario (similar to the way Travis-CI's `matrix` feature works)